README.md

```diff
@@ -2,6 +2,9 @@
 
 Preview components in isolation with documentation, annotations and breakpoint controls.
 
+## Live preview
+
+* [Bootstrap 3](https://voorhoede.github.io/bootstrap-v3)
```

Comment:
This URL won't work. Guessing it should be `https://voorhoede.github.io/demo-viewer/bootstrap-v3/`. Better yet, I'd like it to be `https://voorhoede.github.io/demo-viewer/examples/bootstrap-v3/` (will elaborate in general comment).